Kitodo/src/main/webapp/WEB-INF/templates/includes/massImport/fileMassImport.xhtml

```diff
@@ -23,11 +23,11 @@
         <p:outputLabel for="catalogueSelect"
                        value="#{msgs['newProcess.catalogueSearch.catalogue']}"/>
         <p:selectOneMenu id="catalogueSelect"
-                         immediate="true"
                          value="#{MassImportForm.selectedCatalog}">
             <f:selectItem itemValue="#{null}" itemLabel="-- #{msgs.selectCatalog} --" noSelectionOption="true"/>
-            <f:selectItems value="#{CreateProcessForm.importTab.catalogs}" var="step" itemLabel="#{step}"
-                           itemValue="#{step}"/>
+            <f:selectItems value="#{CreateProcessForm.importTab.catalogs}" var="catalog" itemLabel="#{catalog}"
```

Comment:
Would it be possible to deactivate the **Upload** button until a valid OPAC is selected? At the moment, selecting a CSV file and clicking the **Upload** button without selecting a catalog causes a `ConfigException` in the backend but no feedback is send to the user on the frontend:
```
org.kitodo.exceptions.ConfigException: The catalog 'null' is not found!
	at org.kitodo.config.OPACConfig.getCatalog(OPACConfig.java:226)
```